test/e2e/common/e2e_config/e2e_config.go

```diff
@@ -95,31 +99,68 @@ func configFileExists(path string) bool {
 // so we cannot use logf or Expect instead we use fmt.Print... and panic.
 func GetConfig() E2EConfig {
 	var err error
+	e2eRootDir, okE2eRootDir := os.LookupEnv("e2e_root_dir")
+	// The configuration overrides the e2eRootDir setting,
+	// this makes it possible to use a configuration file written out
+	// previously to replicate a test run configuration.
 	once.Do(func() {
+		var configFile string
 		// We absorb the complexity of locating the configuration file here
 		// so that scripts invoking the tests can be simpler
-		// - if OS envvar e2e_config is not defined the config file is defaulted to ci_e2e_config
 		// - if OS envvar e2e_config is defined
 		//		- if it is a path to a file then that file is used as the config file
 		//		- else try to use a file of the same name in the configuration directory
-		configFile := fmt.Sprintf("%s/ci_e2e_config.yaml", configDir)
-		// A configuration file *MUST* be provided.
+		// - Otherwise the config file is defaulted to ci_e2e_config
+		// A configuration file *MUST* be specified.
```

Comment:
Is this comment correct? It seems you don't have to specify the config file - there is a default file.
But the config file still needs to exist - the old comment was right?